tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -675,7 +675,9 @@ def test_flush_queue_is_thread_safe():
     from mlflow.entities import Metric
     from mlflow.tensorflow import _flush_queue, _metric_queue_lock
 
-    metric_queue_item = ("run_id1", Metric("foo", "bar", 100, 1))
+    client = mlflow.tracking.MlflowClient()
+    run = client.create_run(experiment_id="0")
+    metric_queue_item = (run.info.run_id, Metric("foo", 0.1, 100, 1))
```

Comment:
Fix for this error:

https://github.com/mlflow/mlflow/runs/1891436216#step:5:321

```
E           mlflow.exceptions.MlflowException: Got invalid value bar for metric 'foo' (timestamp=100). Please specify value as a valid double (64-bit floating point)
```

and

https://github.com/mlflow/mlflow/runs/1891792593#step:5:331

```
https://github.com/mlflow/mlflow/runs/1891792593#step:5:331
```